RecentActivity/src/org/sleuthkit/autopsy/recentactivity/RAImageIngestModule.java

```diff
@@ -85,6 +85,7 @@ public void startUp(IngestJobContext context) throws IngestModuleException {
         Extract webAccountType = new ExtractWebAccountType();
         Extract messageDomainType = new DomainCategoryRunner();
 
+        extractors.add(recycleBin); // this needs to run after ExtractRegistry and ExtractOS
```

Comment:
@markmckinnon the comment says that it needs to run after ExtractRegistry.  Is that no longer true?